app/views/collections/index.html.erb

```diff
@@ -21,20 +21,27 @@
   <h1>Collections</h1>
 
   <form action="/collections" method="POST" class='new-collection-container' >
-    <label for="title">Create Collection</label><br>
-    <input type="text" id="title" name="title" class='new-collection-input' placeholder='name of collection...'> <br>
-    <input type="submit" value="Save" class='new-collection-button'>
-  </form> 
-
-  <br>
-  <br>
+    <label class="create-collection-form-heading" for="title">Create Collection</label><br>
+    <input type="text" id="title" name="title" maxlength='20' class='new-collection-input' placeholder='name of collection...'> <br>
+    <button type="submit" disabled value="Save" class='new-collection-button'>save</button>
+  </form>
+
+  <script type="text/javascript" defer>
+    const validateInputs = () => {
+      if (document.querySelector('.new-collection-input').value !== '') {
+        document.querySelector('.new-collection-button').disabled = false;
+      } else if (document.querySelector('.new-collection-input').value === '') {
+        document.querySelector('.new-collection-button').disabled = true;
```

Comment:
Given that the form submit happens on clicking the `save` button, I like how we were able to disable the button so the form won't refresh.